CONTRIBUTING.md

```diff
@@ -1,11 +1,26 @@
 # Contribute to virt-manager
 
+## Setting up the development environment
+
+This project uses `meson` and `cmake` as build system and `python3` as programming language.
+In order to keep code styling consistent the project adopts `pylint`, `pycodestyle`
+and `codespell`. For testing, `pytest` has been choosen as testing framework.
+For localizing the project `gettext` is being used.
+The graphic interfaces is based on `gtk4`, `glib2` and `gobject-introspection`. The build system relies on `pkgconf`
+for detecting them.
+
+In order to get what's needed on a Fedora or derivative distribution you can install everything with:
+
+```console
+dnf install -y meson python3-devel gtk4 glib2-devel pytest pylint codespell pkgconf gobject-introspection
+```
+
```

Comment:
This section is not correct, we are still on GTK 3 and we don't use cmake at all. In addition this basically duplicates what is in INSTALL.md file.